packages/executor/tsconfig.json

```diff
@@ -1,11 +1,27 @@
 {
-  "extends": "../../tsconfig.json",
-
   "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "sourceMap": true,
+    "esModuleInterop": true,
+    "composite": true,
+    "resolveJsonModule": true,
+    "declaration": true,
+    "noImplicitAny": false,
+    "removeComments": true,
+    "noLib": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "typeRoots": [
+      "node_modules/@types"
+    ],
     "rootDir": "./src",
     "outDir": "./dist"
   },
-
+  "exclude": [
+    "node_modules",
+    "dist"
+  ],
```

Comment:
Why can't we just extend the root tsconfig? Does it have to do with the docker container?

Reply:
Yeah, we need the full config to be copied into the docker container for it to work correctly. 

There probably ways to avoid the duplication here like copying both the root config and the package config or combining them together and copying the result into the container, but I didn't think it was worth worrying about at the moment.